app/pathogen/arbovirus/visualizations/page.tsx

```diff
@@ -24,7 +23,7 @@ export default function VisualizationsPage() {
   }
 
   return (
-    <div className="w-screen overflow-y-hidden grid grid-cols-12 grid-rows-2 -my-4 -ml-4 h-full-screen">
+    <div className="w-screen overflow-y-hidden grid grid-cols-12 grid-rows-2 h-full-screen">
```

Comment:
God bless lol. This was removed because the `p-4` that was needed for the old dashboard and analyze page was also removed. This was to offset that class basically.